src/hst_autocompile.sh

```diff
@@ -608,6 +609,62 @@ if [ "$PHP_B" = true ]; then
 	fi
 fi
 
+#################################################################################
+#
+# Building hestia-web-terminal
+#
+#################################################################################
+
+if [ "$WEB_TERMINAL_B" = true ]; then
+	if [ "$CROSS" = "true" ]; then
```

Comment:
I am sure this is not really OS depended as it runs on Nodejs anyway

Reply:
I honestly didn't check the cross variable before putting this there. I'll remove it:)

Reply:
Actually nevermind, node-pty needs to be compiled when it gets installed